src/io/camera.js

```diff
@@ -1,16 +1,18 @@
 // This file was adapted from infragram-js:
 // http://github.com/p-v-o-s/infragram-js.
 module.exports = function Camera(options) {
-  var canvas, 
-      ctx;
+  var canvas,
+  ctx;
 
   // Initialize getUserMedia with options
   function initialize() {
     getUserMedia(webRtcOptions, success, deviceError);
 
     // iOS Safari 11 compatibility: https://github.com/webrtc/adapter/issues/685
-    webRtcOptions.videoEl.setAttribute('autoplay', 'autoplay');
-    webRtcOptions.videoEl.setAttribute('playsinline', 'playsinline');
+    webRtcOptions.videoEl.setAttribute('id', 'webCamVideoEl');
+    var webCamVideoEl = document.getElementById('webCamVideoEl')
```

Comment:
Here also we could set this from the constructor options! See my next comment below.

Reply:
Please help me with the follow up

Reply:
Please help me with the follow up comment.

Reply:
```suggestion
    options.webcamVideoSelector = options.webcamVideoSelector || 'webcamVideoEl';
    options.webcamVideoEl = document.getElementById(options.webcamVideoSelector);
    webRtcOptions.videoEl.setAttribute('id', options.webcamVideoSelector);
```

How about this? So we can set it explicitly in the constructor, but it defaults to `webcamVideoEl`?

Also, in another PR, we add to the README to show exactly what can be set in the constructor. We can add a section to https://github.com/publiclab/infragram/#contributing -- following the general format of this example: https://github.com/publiclab/inline-markdown-editor/#usage

Does that make sense? What do you think?